OnionAppWithCQRS/Application/CQRS/Commands/UpdateOfficeCommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Domain;
+using MediatR;
+
+namespace Application.CQRS.Commands
+{
+    public class UpdateOfficeCommand : IRequest<Office>
+    {
+        public int OfficeId { get; set; }
```

Comment:
Obviously, this is an office Id, you can just call it an Id

Reply:
fix